dist/screeps.d.ts

```diff
@@ -1017,8 +1017,7 @@ interface SignDefinition {
 }
 interface StoreDefinition {
     [resource: string]: number;
-    energy?: number;
-    power?: number;
+    energy: number;
```

Comment:
While ```typescript
interface StoreDefinition {
    [resource: string]: number | undefined;
    energy: number;
}```

Is the most correct representation, it doesn't capture that `store.snafucate` is infact never of type `number` but always of type `undefined` (Unless we invent the `snafucate` resource).

Something like ```typescript
interface StoreDefinition {
    [resource: keyof Resources]: number | undefined;
    energy: number;
}```

Would be the most correct, assuming a type called `Resources` compatible with the `keyof` operator in TS 2.1.